styles/buttons.css

```diff
@@ -0,0 +1,56 @@
+@import url(./main.css);
+/* ------------- */
+/* Buttons */
+/* ------------- */
+
+.primary-btn {
+  background-color: hsl(var(--bee-yellow));
+  padding: 0.7rem 1rem;
+  margin: 1rem 1rem;
```

Comment:
Suggestion: margin:1rem will affect the same.

Reply:
right 😆 thanks!